lib/gaffe/version.rb

```diff
@@ -1,3 +1,3 @@
 module Gaffe
-  VERSION = '1.0.2'
+  VERSION = '1.0.2'.freeze
```

Comment:
Yes, but this makes rubocop happy. Otherwise travis build fails because of rubocop